static/views/edit-developer-profile.html

```diff
@@ -185,6 +264,46 @@ <h3>Edit Your Developer Profile</h3>
             {{ end }}
         </p>
         <input type="email" name="email" disabled id="email" placeholder="Your Email" style="width: 100%;" value="{{ .DeveloperProfile.Email }}"><br>
+        <span id="experience">Experience <div onclick="openDeveloperMetadataModal('experience');" style="float:right; cursor: pointer;"></span>Add</div> <br />
```

Comment:
- div should not be inside a span
- span is broken, you open span, open div, close span, close div, this is broken if you look at the browser source

can you fix this broken html?